tests/javascript/core/spec.js

```diff
@@ -244,4 +244,72 @@ define(['jquery', 'testsRoot/core/spec-setup', 'jasmineJquery'], function ($) {
 			expect(this.form.filter_order_Dir.value).toEqual('dir')
 		});
 	});
+
+	describe('Core Joomla.submitbutton', function () {
+		var form = document.querySelectorAll( 'form.form-validate' );
+		form.task = {};
+
+		beforeEach(function () {
+			spyOnEvent('#adminForm', 'submit');
+			form.removeChild = jasmine.createSpy('removeChild');
+			$('#applyBtn').click();
+		});
+
+		it('should form value greater than 0', function () {
+			expect(form.length).toBeGreaterThan(0);
+		});
+	});
+
+	describe('Core Joomla.extend', function () {
+		var destination;
+		var source;
+		beforeEach(function () {
+			destination = {};
+			source = {
+				p: 'value',
+				p1: 'value1'
+			};
+		});
+
+		beforeAll(function () {
+			Joomla.extend(destination, source);
+		});
+
+		it('should have a property with the name of the argument', function () {
+			expect(source).toEqual(jasmine.objectContaining({
+				p: 'value'
+			}));
+			expect(source).not.toEqual(jasmine.objectContaining({
+				p2: 'value2'
+			}));
+		});
+	});
+
+	describe('Core Joomla.ajaxErrorsMessages', function () {
+		var xhr = new XMLHttpRequest(),
+			textStatus = ["parsererror", "nocontent", "timeout", "abort"],
+			error = ["Error one", "Error two", "Error three", "Error four"];
+
+		beforeAll(function () {
+			Joomla.ajaxErrorsMessages(xhr, textStatus, error);
+		});
+
+		it ('should have textStatus = parsererror for textStatus value', function () {
+			expect(textStatus[0]).toEqual('parsererror');
```

Comment:
Here all the test cases verifies the values of array defined above which doesn't add a clear value for the test coverage of Core

var textStatus = ["parsererror", "nocontent", "timeout", "abort"]

Test textStatus[0] to be "parsererror"
Test textStatus[1] to be "nocontent" & etc.

Reply:
Thanks for pointing it out. Need to fix the issue